contribute/ISSUE_TRIAGE.md

```diff
@@ -359,6 +362,5 @@ This will give you a structure of labels in the sidebar similar to the following
   - Grafana
 ```
 
-- All notifications you’ll need to read/take action on show up as unread in GitHub (mine) and its sub-labels.
-- All other notifications you don’t need to take action on show up as unread in GitHub (other) and its sub-labels
-  - This is convenient for issue triage and to follow the activity in the Grafana project.
+- All notifications you’ll need to take action on should show up as unread in GitHub (mine) and its sub-labels.
+- All notifications you don’t need to take action on show up as unread in GitHub (other) and its sub-labels.
```

Comment:
Because it's a UI element in your inbox.

```suggestion
- All notifications you’ll need to take action on should show up as unread in **GitHub (mine)** and its sub-labels.
- All notifications you don’t need to take action on show up as unread in **GitHub (other)** and its sub-labels.
```